### PR TITLE
Update in.m3u

### PR DESCRIPTION
**Pull request by vishveshjain**
Added all sony network channels

### CODE REVIEW

**Review by Alstruit — CHANGES_REQUESTED** [review]
[CONTRIBUTING.md#stream-description-scheme](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#stream-description-scheme)

**Review by vishveshjain — COMMENTED** [review]
Changes made as per [iptv](https://github.com/iptv-org/iptv/tree/master)
/CONTRIBUTING.md

File: streams/in.m3u
Comment (Alstruit) [inline streams/in.m3u:529]:
```suggestion
#EXTINF:-1 tvg-id="SonyEntertainmentTelevisionAsiaHD.in",Sony Entertainment Television Asia HD (1080p)
```

File: streams/in.m3u
Comment (Alstruit) [inline streams/in.m3u:531]:
```suggestion
#EXTINF:-1 tvg-id="SonySAB.in",Sony SAB (1080p)
```

**Review by Alstruit — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Comment by vishveshjain** [comment]
Hi still waiting for 1 more approval.

**Review by BellezaEmporium — APPROVED** [review]
LGTM.

**Pull request merged**
